modules/tink_cc/2.1.2/presubmit.yml

```diff
@@ -1,10 +1,12 @@
-build_targets: &build_targets
-- '@tink-cc//tink:tink_cc'
-
-platforms:
-  debian10:
-    build_targets: *build_targets
-  macos:
-    build_targets: *build_targets
-  ubuntu2004:
-    build_targets: *build_targets
+matrix:
+  platform:
+  - ubuntu2004
+  - macos
+  bazel: [6.x]
```

Comment:
Any reason to not support 7.x here?

Reply:
Does it work on Windows?

Reply:
Added both. In our internal CI we do test Windows but not Bazel 7.x; I did a quick test and I didn't see issues with Bazel 7.x anyhow so enabling it.